store/tikv/txn.go

```diff
@@ -250,7 +250,11 @@ func (txn *KVTxn) Commit(ctx context.Context) error {
 	}
 	defer func() {
 		// For async commit transactions, the ttl manager will be closed in the asynchronous commit goroutine.
-		if !committer.isAsyncCommit() {
+		if committer.isAsyncCommit() || committer.isOnePC() {
```

Comment:
@sticnarf @youjiali1995 

Could we just always close the ttl manager here, seems at commit phase and the async commit/1pc transaction size will be small.

Reply:
I think it is okay. The chance is small that TTL expires before the primary lock is committed.

Reply:
I think yes because `SafeWindow`(2s) is smaller than the interval of keepAlive(10s). It's nearly useless here.

Reply:
It seems to me not related to `SafeWindow`. `SafeWindow` only constrains the prewrite time.